packages/Toast/src/Toast.styles.js

```diff
@@ -0,0 +1,100 @@
+import styled, { css } from "styled-components";
+import tinycolor from "tinycolor2";
+
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+import Button from "@paprika/button";
+import { visuallyHidden } from "@paprika/stylers/lib/includes";
+
+function getStylesForType(type) {
+  if (type === "visually-hidden") return "";
+  const primaryColors = {
+    success: tokens.color.greenLighten50,
+    warning: tokens.color.yellowLighten30,
+    error: tokens.color.orangeLighten40,
+    info: tokens.color.cremeLighten5,
+    locked: tokens.color.yellowLighten30,
+  };
+
+  return css`
+    background-color: ${primaryColors[type]};
+    border-color: ${tinycolor(primaryColors[type])
+      .darken(10)
```

Comment:
Interesting... 🤔 I'm concerned about including a whole library just for this function... but then again, lighten and darken functions would be really nice to have access to since we can use the ones from Sass.  What if we expose both of those via the `stylers` package, so we can hide the implementation details a bit?  There are some nice tools in that `tinycolor` library we might want to use later, like `setAlpha()` and `mostReadable()`.

Reply:
We can also hardcode those colors since they're only used in toast...

Reply:
That is the simpler solution! If you are up to it, you could create a "Feature Request" in issues to use this library to provide `lighten`, `darken`, `setAlpha` and any others you think would be particularly useful.